### PR TITLE
Update for python math.js

### PR DESCRIPTION
**Pull request by LouisCatala**
## The basics

- [ ] I branched from develop
- [ ] My pull request is against develop
- [ ] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### CODE REVIEW

File: generators/python/math.js [hunk 1]
```diff
@@ -157,67 +157,70 @@ Blockly.Python['math_constant'] = function(block) {
...
+     '  for x in range(6, int(math.sqrt(n)) + 2, 6):',
+     '    if n % (x - 1) == 0 or n % (x + 1) == 0:',
+     '      return False',
+     '  return True']);
+    code = functionName + '(' + numberToCheck + ')';
+  } else if (dropdownProperty === 'DIVISIBLE_BY') {
+    const divisor = Blockly.Python.valueToCode(block, 'DIVISOR',
+    Blockly.Python.ORDER_MULTIPLICATIVE) || '0';
+    // If 'divisor' is some code that evals to 0, Python will raise an error.
+    if (!divisor || divisor == '0') {
```
Comment (RichDoherty) [inline generators/python/math.js]: Change `==` to `===`.

File: generators/python/math.js [hunk 2]
```diff
@@ -157,67 +157,70 @@ Blockly.Python['math_constant'] = function(block) {
...
+     '    return False',
+     '  # Check all the numbers of form 6k +/- 1, up to sqrt(n).',
+     '  for x in range(6, int(math.sqrt(n)) + 2, 6):',
+     '    if n % (x - 1) == 0 or n % (x + 1) == 0:',
+     '      return False',
+     '  return True']);
+    code = functionName + '(' + numberToCheck + ')';
+  } else if (dropdownProperty === 'DIVISIBLE_BY') {
+    const divisor = Blockly.Python.valueToCode(block, 'DIVISOR',
+    Blockly.Python.ORDER_MULTIPLICATIVE) || '0';
```
Comment (RichDoherty) [inline generators/python/math.js]: Blockly.Python.ORDER_MULTIPLICATIVE) || '0'; needs to be indented.

File: generators/python/math.js [hunk 3]
```diff
@@ -157,67 +157,70 @@ Blockly.Python['math_constant'] = function(block) {
...
+  }
+  const dropdownProperty = block.getFieldValue('PROPERTY');
+  const tuple = PROPERTIES[dropdownProperty];
+  const suffix = tuple[0];
+  const inputOrder = tuple[1];
+  const outputOrder = tuple[2];
+  const numberToCheck = Blockly.Python.valueToCode(block, 'NUMBER_TO_CHECK',
+      inputOrder) || '0';
+  let code;
+  if (dropdownProperty == 'PRIME') {
```
Comment (RichDoherty) [inline generators/python/math.js]: Change `==` to `===`.

**Review by RichDoherty — CHANGES_REQUESTED** [review]

File: generators/python/math.js [hunk 4]
```diff
@@ -157,67 +157,71 @@ Blockly.Python['math_constant'] = function(block) {
...
+       '  for x in range(6, int(math.sqrt(n)) + 2, 6):',
+       '    if n % (x - 1) == 0 or n % (x + 1) == 0:',
+       '      return False',
+       '  return True']);
+       code = functionName + '(' + numberToCheck + ')';
+  } else if (dropdownProperty === 'DIVISIBLE_BY') {
+    const divisor = Blockly.Python.valueToCode(block, 'DIVISOR',
+    Blockly.Python.ORDER_MULTIPLICATIVE) || '0';
+    // If 'divisor' is some code that evals to 0, Python will raise an error.
+    if (!divisor || divisor === '0') {
```
Comment (jeremyjacob123) [inline generators/python/math.js]: `!divisor` is not needed, as it will always be `false`.
```suggestion
    if (divisor === '0') {
```

**Review by jeremyjacob123 — CHANGES_REQUESTED** [review]

**Review by jeremyjacob123 — APPROVED** [review]

**Review by RichDoherty — APPROVED** [review]

File: generators/python/math.js [hunk 5]
```diff
@@ -184,31 +184,32 @@ Blockly.Python['math_number_property'] = function(block) {
...
+       '  # False if n is negative, is 1, or not whole,' +
+       ' or if n is divisible by 2 or 3.',
+       '  if n <= 1 or n % 1 != 0 or n % 2 == 0 or n % 3 == 0:',
+       '    return False',
+       '  # Check all the numbers of form 6k +/- 1, up to sqrt(n).',
+       '  for x in range(6, int(math.sqrt(n)) + 2, 6):',
+       '    if n % (x - 1) == 0 or n % (x + 1) == 0:',
+       '      return False',
+       '  return True']);
+       code = functionName + '(' + numberToCheck + ')';
```
Comment (jschanker) [inline generators/python/math.js:211]: `code` should only be indented four spaces to line up with `const functionName` above.

File: generators/python/math.js [hunk 6]
```diff
@@ -157,67 +157,70 @@ Blockly.Python['math_constant'] = function(block) {
...
+        Blockly.Python.ORDER_RELATIONAL],
+    'POSITIVE': [' > 0', Blockly.Python.ORDER_RELATIONAL,
+        Blockly.Python.ORDER_RELATIONAL],
+    'NEGATIVE': [' < 0', Blockly.Python.ORDER_RELATIONAL,
+        Blockly.Python.ORDER_RELATIONAL],
+    'DIVISIBLE_BY': [null, Blockly.Python.ORDER_MULTIPLICATIVE,
+        Blockly.Python.ORDER_RELATIONAL],
+    'PRIME': [null, Blockly.Python.ORDER_NONE,
+        Blockly.Python.ORDER_FUNCTION_CALL]
+  }
```
Comment (jschanker) [inline generators/python/math.js:176]: A semicolon is required here to conform to Google's style guide.

**Pull request merged**